src/usr/local/containerbase/tools/v2/poetry.sh

```diff
@@ -3,6 +3,10 @@
 # shellcheck source=/dev/null
 . "$(get_containerbase_path)/utils/python.sh"
 
+function install_tool () {
+  install_python_tool
+}
+
```

Comment:
move this to utils/python.sh as default, so it only needs to be over written where needed.